python/ql/src/semmle/python/pointsto/PointsTo.qll

```diff
@@ -694,7 +694,7 @@ module PointsToInternal {
 
     /* Holds if `import name` will import the module `m`. */
     cached
-    predicate module_imported_as(ModuleObjectInternal m, string name) {
+    deprecated predicate module_imported_as(ModuleObjectInternal m, string name) {
         /* Normal imports */
         m.getName() = name
```

Comment:
nitpick: Can the body of this method be replaced with `module_imported_as_with_context(m, name, _)`? (that would be a bit more clean in my opinion)